DNN Platform/DotNetNuke.Web.Deprecated/UI/WebControls/DnnDatePicker.cs

```diff
@@ -33,11 +33,6 @@ public class DnnDatePicker : RadDatePicker
     {
         protected override void OnInit(EventArgs e)
         {
-			if (CultureInfo.CurrentCulture.Name == "ar-SA")
```

Comment:
This is fixed, but I'm not sure why git didn't track the change and remove your comment like the others.... I'm guessing it's because of the indentation?